src/reefState/index.ts

```diff
@@ -1,48 +1,26 @@
-export { initReefState } from "./initReefState";
-export type { StateOptions } from "./initReefState";
-export {
-  toInjectedAccountsWithMeta,
-  accounts_status$,
-} from "./account/accounts";
-export {
-  accounts$,
-  selectedAccount$,
-  selectedNFTs$,
-  selectedTokenBalances$,
-  selectedTokenPrices$,
-} from "./selected-account-plain-data.rx";
-export {
-  selectedAccount_status$,
-  selectedAddress$,
-} from "./account/selectedAccount";
-export { onTxUpdateResetSigners } from "./account/accountsLocallyUpdatedData";
-export { selectedAccountAddressChange$ } from "./account/selectedAccountAddressChange";
-export { setSelectedAddress, setAccounts } from "./account/setAccounts";
-export {
-  selectedTokenBalances_status$,
-  // availableReefPools_status$,
-  selectedNFTs_status$,
-  selectedPools_status$,
-  selectedTokenPrices_status$,
-  selectedTransactionHistory_status$,
-} from "./tokenState.rx";
-export { setSelectedNetwork, selectedNetwork$ } from "./networkState";
-export {
-  instantProvider$,
-  selectedProvider$,
-  selectedNetworkProvider$,
-  providerConnState$,
-} from "./providerState";
-export {
-  FeedbackStatusCode,
-  StatusDataObject,
-  isFeedbackDM,
-  findMinStatusCode,
-  skipBeforeStatus$,
-} from "./model/statusDataObject";
-export type { FeedbackStatus } from "./model/statusDataObject";
-export type { UpdateAction } from "./model/updateStateModel";
-export { UpdateDataType } from "./model/updateStateModel";
-export { addPendingTransactionSubj, pendingTxList$ } from "./tx/pendingTx.rx";
-
-export { reloadTokens } from "./token/force-reload-tokens";
+export * from "./account/accountStateUtil";
```

Comment:
let's export only what's needed - the more we expose from lib the more we are stuck with it - so export from lib specific references we need elsewhere and no more